service-runner/src/main/resources/config/reference-config.yaml

```diff
@@ -33,10 +33,10 @@ auth:
   # CallbackAPI requests (`Platform->Anchor`) will contain an authentication header whose token was built using this
   # secret. The Anchor Backend will also store this same secret and use it to decode the incoming token to verify it
   # came from the Platform.
-  platformToAnchorSecret: myPlatformToAnchorSecret
+  platformToAnchorSecret: myPlatformToAnchorSecret1234567890
```

Comment:
What's the purpose of the trailing `1234567890`?

Reply:
Make key seize 256 bit